js/request.js

```diff
@@ -150,7 +150,6 @@ exports.request = function(req, res, opts, handler) {
       var terminationMessage = 'Terminating server at http://[localhost]:' + opts.port + ' for ' + opts['module-path'] + ' / ' + opts.handler;
       res.end(terminationMessage + '\n');
       console.info(terminationMessage);
-      server.close();
```

Comment:
This line should terminate the http server thus allowing the process to terminate. This is the entire point of the DELETE request. If you remove this line, then you may as well remove the entire handling of DELETE method requests.

It appears that `server` is defined in `startup.js` and not passed through. Ensuring that the DELETE function has access to the `server` would be a better fix.